### DISCUSSION TITLE
Bug: Salesforce Component doesnot have a way to pass login details or client

### DISCUSSION THREAD

**Discussion by Premcnathan** [apache/camel-karavan#1464] [Q&A] [answered]
### Describe the bug

For using salesforce componet in karavan, there is no provision to provide Salesforce login (security)details like login URl, keystore, password etc. 

To take an example, AWS S3 component has "Component security properties" where such properties can be added.

Is there a way we can pass that infroamtion to Saleforce componet?

### Steps to reproduce the behavior

1. Go to karavan.space
2. Choose Salesforce component.

### Variant

Vs Code Extension

### Container Management (if applicable)

None

### Operating System (if applicable)

None

### Version

4.7.0

**Comment by davsclaus** [comment, marked as answer]
You configure this in the salesforce component level.
Alternative is to use the salesforce source kamelet that allows to configure this in karavan
